scripts/update_rn_tester.sh

```diff
@@ -25,3 +25,13 @@ update_pods () {
   bundle exec pod install
   cd "$THIS_DIR" || exit
 }
+
+update_package_json () {
+  if [ $# -eq 0 ]
+  then
+    exit
+  fi
+  cd "$RNTESTER_DIR" || exit
+  sed -i '' "s/1000.0.0/$1/g" package.json || exit
```

Comment:
`sed` has different syntax on macOS compared to the one installed in other OSes:
```suggestion
  if sed --version &> /dev/null; then
    sed -i'' "s/1000.0.0/$1/g" package.json || exit
  else
    sed -i '' "s/1000.0.0/$1/g" package.json || exit
  fi
```
If we can assume `jq` is installed, it's a better alternative.

Reply:
Took the change you suggested. I can probably "brew install JS" if needed, but I see from https://github.com/facebook/react-native/pull/42730 that there might be a desire to drop unneeded dependencies. I'll let whoever reviews this from Meta chime in on what they prefer.